OrbitGgp/include/OrbitGgp/GgpSshInfo.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBITGGP_GGP_SSH_INFO_H_
+#define ORBITGGP_GGP_SSH_INFO_H_
+
+#include <QJsonDocument>
```

Comment:
This include is not needed here, right? But instead we should include QByteArray.

Reply:
true